### ISSUE TITLE
evalution results in human3.6M?

### ISSUE THREAD

**Issue by qiuzhongwei-USTB** [karfly/learnable-triangulation-pytorch#44]
Hi,

I saw that you use every fifth frame for evaluation in your paper.
Is it means that the MPJPE of 20.8 is evaluated on every fifth frame?

Have you evaluated your model on the full frames? Could you provide the results on the full frames?
Thanks

**Comment by karfly** [comment]
Hi, @qiuzhongwei-USTB.

We found out that it doesn’t affect the resulting accuracy a lot.

Before the commit #42 in this repository was done on all frames resulting in 20.5 mm MPJPE error. Honestly, the weights of the pretrained model we uploaded to the repository were not the ones we used in the paper (current model was trained for a longer time). That’s why the error is lower.
